pac4j-core/src/main/java/org/pac4j/core/client/DirectClient.java

```diff
@@ -32,18 +33,18 @@ protected final void internalInit() {
     protected abstract void clientInit();
 
     @Override
-    public final HttpAction redirect(final WebContext context) {
-        throw new TechnicalException("direct clients do not support redirections");
+    public final Optional<HttpAction> redirect(final WebContext context) {
+        return Optional.empty();
```

Comment:
Let's keep the old code here or let's try to move this method in the `IndirectClient`?